hack/examples/sync.sh

```diff
@@ -19,93 +19,26 @@ mkdir -p "${EXAMPLE_CLUSTERCLASSES_DIR}"
-                                        and .spec.topology.variables[0].value.addons.cni.strategy == "HelmAddon"
-                                      )' >"${EXAMPLE_CLUSTERS_DIR}/docker-cluster-calico-helm-addon.yaml"
-    ) \
-    >(
-      gojq --yaml-input --yaml-output 'select(.metadata.labels["cluster.x-k8s.io/provider"] == "aws"
-                                        and .kind != "Cluster"
-                                        and .kind != "AWSClusterStaticIdentity"
-                                      )' >"${EXAMPLE_CLUSTERCLASSES_DIR}/aws-cluster-class.yaml"
-    ) \
-    >(
-      gojq --yaml-input --yaml-output 'select(.metadata.labels["cluster.x-k8s.io/provider"] == "aws"
-                                        and .kind == "Cluster"
-                                        and .spec.topology.variables[0].value.addons.cni.provider == "Calico"
-                                        and .spec.topology.variables[0].value.addons.cni.strategy == "ClusterResourceSet"
-                                      )' >"${EXAMPLE_CLUSTERS_DIR}/aws-cluster-calico-crs.yaml"
-    ) \
-    >(
-      gojq --yaml-input --yaml-output 'select(.metadata.labels["cluster.x-k8s.io/provider"] == "aws"
-                                        and .kind == "Cluster"
-                                        and .spec.topology.variables[0].value.addons.cni.provider == "Calico"
-                                        and .spec.topology.variables[0].value.addons.cni.strategy == "HelmAddon"
-                                      )' >"${EXAMPLE_CLUSTERS_DIR}/aws-cluster-calico-helm-addon.yaml"
-    ) \
-    >(
-      gojq --yaml-input --yaml-output 'select(.metadata.labels["cluster.x-k8s.io/provider"] == "aws"
-                                        and .kind == "Cluster"
-                                        and .spec.topology.variables[0].value.addons.cni.provider == "Cilium"
-                                        and .spec.topology.variables[0].value.addons.cni.strategy == "ClusterResourceSet"
-                                      )' >"${EXAMPLE_CLUSTERS_DIR}/aws-cluster-cilium-crs.yaml"
-    ) \
-    >(
-      gojq --yaml-input --yaml-output 'select(.metadata.labels["cluster.x-k8s.io/provider"] == "aws"
-                                        and .kind == "Cluster"
-                                        and .spec.topology.variables[0].value.addons.cni.provider == "Cilium"
-                                        and .spec.topology.variables[0].value.addons.cni.strategy == "HelmAddon"
-                                      )' >"${EXAMPLE_CLUSTERS_DIR}/aws-cluster-cilium-helm-addon.yaml"
-    ) \
-    >/dev/null
-
-#shellcheck disable=SC2016
-sed -i'' 's/^  name: .\+$/  name: ${CLUSTER_NAME}/' "${EXAMPLE_CLUSTERS_DIR}"/*.yaml
+kustomize build ./hack/examples/overlays/clusterclasses/aws >"${EXAMPLE_CLUSTERCLASSES_DIR}/aws-cluster-class.yaml"
```

Comment:
line 22-23 and 26 are doing something similar?

Reply:
Oops yeah I forgot to remove these lines when I moved them into the for loop. Will fix up tomorrow. Thanks!

Reply:
Done! Thanks again for pointing this out.